<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editar_motorista</class>
 <widget class="QWidget" name="editar_motorista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>641</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>CNPJ/CPF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_cpf_motorista"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Razão social</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_razao_social"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Nome fantasia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_nome_motorista"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Endereço</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_endereco"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>DDD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_ddd"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Telefone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_telefone"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cadastro_motorista">
        <property name="text">
         <string>Cadastrar Motorista</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
